cmp/form/SliderField.js

```diff
@@ -24,6 +25,8 @@ import {HoistField} from './HoistField';
 export class SliderField extends HoistField {
```

Comment:
Sorry -- Just noticed this.  Seems very weird to me that just one HoistField would have layoutSupport.

Reply:
Yes, by the default the slider always takes 100% of the room available and applying styles to it has no effect. So I had to wrap it with a box to make it work, otherwise the labels would get cut off. Maybe it's blueprintjs issue.
Spent sometime with @TomTirapani  on this problem this morning

Reply:
yes, but could we do that workaround internally to the control?  Why does the workaround require it to implement LayoutSupport